HanekeSwift.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name = 'HanekeSwift'
   s.module_name = 'Haneke'
-  s.version = '0.10.1'
+  s.version = '0.10.2'
```

Comment:
The podspec version should be the same of the project version. the issue here is that pod support has been discontinued after 0.10.2. If you wanted latest commits you had to refer to specific branches or straight to master. this is not a best practice and a new pod version should be released. Good 1.1 as project version, but here too it should change to 1.1

To the maintainer of the project: Please tag master after merge and release a pod. This is something we cannot do, so we need you to do it.